Week-1/Assignment 1.2/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ass-1.2</title>
+ <link rel="stylesheet" href="style.css"> 
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+</head>
+
+<body>
+    <div class="sidebar">
+        <i class="las la-bars"></i></p>
+        <i  class="las la-home"></i></p>
+        <i class="las la-search"></i></p>
+        <i class="las la-volume-up"></i></p>
+        <i class="las la-user"></i></p>
+        <i class="lab la-spotify"></i></p>
+        <i class="lab la-whmcs"></i></p>
+        <i class="lab la-soundcloud"></i>  
+      </div>
+    <header class="header_section">
+        <div class="first-section">
+            <img  src="https://vif1g.csb.app/src/assets/cattyboard.jpg" />
+            <div class="text_img">
+                <div class="headline">
+                    <p id ="head1" text-align: -webkit-center;>CattyBoard Top 100 single charts (11.06.3)</p>
```

Comment:
Lets try our best to have meaningful names to our  attributes to our HTML elements.